src/components/common.js

```diff
@@ -540,7 +540,7 @@ export const ClipboardButton = ({ text, onClick, children, ...props }) => {
   const [copied, setCopied] = useState(false)
   return h(Link, {
     ...props,
-    tooltip: copied ? 'Copied to clipboard' : 'Copy to clipboard',
+    tooltip: props?.tooltip ?? (copied ? 'Copied to clipboard' : 'Copy to clipboard'),
```

Comment:
Can we do away with the new logic if we swap the line `…props,` with the line starting with `tooltip:` since `props.tooltip` would override the default `tooltip` if there exists any `props.tooltip`?